extensions/dicom-seg/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@ohif/extension-dicom-seg",
+  "version": "0.0.1",
+  "description": "OHIF extension for rendering structured reports to HTML",
```

Comment:
Update description

Reply:
Updated. @JamesAPetts maybe you have a better description? i added one based on original pr description.